contrib/babelfishpg_tsql/src/tablecmds.c

```diff
@@ -97,42 +106,56 @@ void pre_check_trigger_schema(List *object, bool missing_ok){
 
 	tgscan = systable_beginscan(tgrel, TriggerRelidNameIndexId, false,
 									NULL, 1, &key);
-
 	while (HeapTupleIsValid(tuple = systable_getnext(tgscan)))
 	{
 		Form_pg_trigger pg_trigger = (Form_pg_trigger) GETSTRUCT(tuple);
-		
-		if (namestrcmp(&(pg_trigger->tgname), depname) == 0)
+		if(!OidIsValid(pg_trigger->tgrelid))
```

Comment:
Why do we have to repeat the logic to find triggers per schema here? What does this function do?

Reply:
I added this function about half year ago, I think it's due to this line : 
```
trigger_schema = ((Value *)list_nth(object,0))->val.str;
```

when in execution time, the find obj hook can't get the trigger_schema refer in the sql 
````
drop trigger trigger_schema.trigger_name
````